src/planscape/plan/views.py

```diff
@@ -568,13 +568,21 @@ def _set_scenario_metadata(priorities, weights, notes, scenario: Scenario):
 
 
 def get_treatment_goals_config(params: QueryDict):
```

Comment:
nit: I'd call this get_treatment_goals_config_for_region so that someone doesn't think it's getting the whole config.